### PR TITLE
Feature 203

### PR DESCRIPTION
**Pull request by Kampi**

### CODE REVIEW

File: .devcontainer/.bashrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export LS_OPTIONS='-F --color=auto'
+alias ls='ls $LS_OPTIONS'
+
+if [ ! -f "~/.zswatch_initialized" ]; then
```
Comment (jakkra) [inline .devcontainer/.bashrc:4]: It's re-running for me every time

**Review by jakkra — COMMENTED** [review]
Changing IP not being picked up
image
image

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,15 +0,0 @@
-{
```
[the remark this answers is not in the document]
Comment (Kampi) [reply .vscode/settings.json:1]: Ah right. It´s used by the CI/CD pipeline, isn´t it?

**Comment by Kampi** [comment]
@jakkra did you change the IP address before building the container? Note that the environment will be copied once the container is built. You can retrigger it by opening the command palette and run `Recreate development container without cache`

File: .devcontainer/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- Install [WSL](https://learn.microsoft.com/en-us/windows/wsl/install)
+- Install the [WSL Plugin for VS Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl)
+- Start VS Code on your host system
+- Start a WSL session in VS Code to let VS Code install all dependencies
+
+## How to use it?
+
+- Start VS Code by executing `code .` in the project root of your Linux instance
+- VS Code is showing a Window with `Reopen in Container` in the bottom right of your screen
+- Reopen the directory in the container
```
[the remark this answers is not in the document]
Comment (Kampi) [reply .devcontainer/README.md:39]: Tested it today and it seems that Docker needs a few seconds to open the shell after the container is up. Can you confirm it?

**Comment by jakkra** [comment]
> @jakkra did you change the IP address before building the container? Note that the environment will be copied once the container is built. You can retrigger it by opening the command palette and run `Recreate development container without cache`

I did not, will try with rebuilding it.

**Comment by Kampi** [comment]
> > @jakkra did you change the IP address before building the container? Note that the environment will be copied once the container is built. You can retrigger it by opening the command palette and run `Recreate development container without cache`
> 
> I did not, will try with rebuilding it.

It´s not a perfect solution IMO because this issue can happen. But I haven´t found a better one and AFAIK you have to set the IP address once because it doesn´t change

**Pull request closed without merging**